internal/v1/api.yaml

```diff
@@ -356,6 +357,10 @@ components:
           items:
             type: string
             example: 'qcow2'
+        repositories:
```

Comment:
May I ask for a short `description` for the new property? Since the API is public for the customers and the spec is the only documentation we have, it would be great to document it as much as possible. Thanks! :)

Reply:
Absolutely, short description added :)